src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBlock.cs

```diff
@@ -12,16 +9,55 @@
 using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml.Documents;
 using Windows.UI.Xaml.Media;
-using Windows.UI.Xaml.Media.Imaging;
 using static Private.Infrastructure.TestServices;
 
+#if __SKIA__
+using SkiaSharp;
+using Windows.UI.Xaml.Documents.TextFormatting;
+#endif
+
 namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls
 {
 	[TestClass]
+	[RunsOnUIThread]
 	public class Given_TextBlock
 	{
+#if __SKIA__
+		[TestMethod]
+		public async Task Check_FontFallback()
```

Comment:
I just tested. The test is passing even without the fix. @jeromelaban I'm not sure how to properly test this :/

Without the fix:

![image](https://github.com/unoplatform/uno/assets/31348972/18adf597-c841-4b75-9851-7f89d82de6cd)

With the fix:

![image](https://github.com/unoplatform/uno/assets/31348972/4508e2a6-5d1a-4166-90aa-495a7ff17495)